extensions/sql/syntaxes/SQL.plist

```diff
@@ -15,10 +15,16 @@
-      			<key>name</key>
-      			<string>keyword.other.sql</string>
+			<key>match</key>
+			<string>(((?<!@)@)|(\[\s*))\b(?i)(\w+)\b</string>
+			<key>name</key>
+			<string>text</string>
```

Comment:
Anything with a leading '@' or '[' should be treated as regular, uncolored text. Is there a better "category" to use here? There's no color theming for "text", so it accomplishes the same goal.